src/plotconfig.jl

```diff
@@ -177,7 +167,10 @@ function PlotConfig(T::Val{:topoplotseries})
             enlarge = 1,
             label_scatter = false,
         ),
-        mapping = (; col = (:time,), row = (nothing,)),
+        mapping = (;
+            col = :time,
```

Comment:
I'm surprised this doesn't need to be a tuple

Reply:
Maybe more consistent if it is a 1-tuple?

Reply:
seems abundant for me

Reply:
but consistent